compiler/rustc_apfloat/src/lib.rs

```diff
@@ -692,4 +692,3 @@ macro_rules! float_common_impls {
 }
 
 pub mod ieee;
-pub mod ppc;
```

Comment:
The funny thing is that the main reason I ported all of it is was so that I wouldn't have to pick and choose tests, and could just run the entire APFloat test suite.

I'm guessing you haven't tried running all the tests on this, otherwise you would've removed a bunch of the tests too.